pmda/test/test_util.py

```diff
@@ -149,22 +149,27 @@ def sumofsquares(a):
     sos : array
         `n x m` array of the sum of squares for 'n' atoms
     """
-    dev = a - np.mean(a, axis=0)
-    sos = np.sum(dev**2, axis=0)
+    dev = a - np.mean(a, axis=0, dtype=np.float64)
+    sos = np.sum(dev**2, axis=0, dtype=np.float64)
     return sos
 
 
 @pytest.fixture(scope="module")
 def pos():
     """Generates array of random positions in range [-100, 100]"""
-    return 200*(np.random.random(size=(100000, 1000, 3)) - 0.5).astype(np.float64)
+    return 200*(np.random.random(size=(100000,
+                                       1000,
+                                       3)) - 0.5).astype(np.float64)
 
 
 @pytest.mark.parametrize('n_frames', [3, 4, 10, 19, 101, 331, 1000])
-def test_second_order_moments(pos, n_frames):
+@pytest.mark.parametrize('isplit', [1, -1, 'other'])
```

Comment:
Good idea.

How about
```python
@pytest.mark.parametrize('isplit', [1, -1] + ["rand{0:03d}".format(i) for i in range(10)])
def test_second_order_moments(pos, n_frames, isplit):
   if isplit.startswith("rand"):
       ....
```

In this way we can easily test 10 (or 100) different random splitting points and get a better chance to catch problematic values.